src/components/order-service/order-service.controller.ts

```diff
@@ -35,9 +44,16 @@ export class OrderServiceController implements angular.IController {
   }
 
   public stepClick(step: any) {
-    if (step.visited) {
-      this.gotoStep(step);
+    // Prevent returning to previous steps if the order is complete.
+    if (this.ctrl.orderComplete) {
```

Comment:
Yeah. We have the problem already for unvisited steps, too. Was holding off trying to fix too many styles since there is some attention needed there